qt/scientific_interfaces/Inelastic/Manipulation/InelasticDataManipulationSymmetriseTabView.h

```diff
@@ -32,16 +32,16 @@ class MANTIDQT_INELASTIC_DLL InelasticDataManipulationSymmetriseTabView : public
   void updateMiniPlots();
   bool validate();
   void setRawPlotWatchADS(bool watchADS);
-  double getEMin();
-  double getEMax();
+  double getElow();
+  double getEhigh();
```

Comment:
We can also change these functions so that they are `const`

Reply:
It's ok. Thanks for the advices.